DenunciaCiudadana/src/components/Complaint.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {View, Text, Image, TouchableOpacity} from 'react-native';
+import {stylesHome} from '../assets/styles/styleHomeDesing';
+
+const Complaint = ({item}) => {
+  return (
+    <View style={stylesHome.complaint}>
+      <View>
+        <Text style={stylesHome.titleComplaint}>Nos cortaron el internet</Text>
+        <View style={stylesHome.nameAndService}>
+          <Text>{item.name}</Text>
+          <Text>{item.area}</Text>
+        </View>
+        <Text>{item.description}</Text>
+      </View>
+      <View>
+        {item.image ? (
+          <Image style={stylesHome.img} source={{uri: `${item.image}`}} />
+        ) : null}
+        {item.image2 ? (
```

Comment:
The name image2 can be better.

```suggestion
        {item.image2 ? (
```

If you have chance I recommend you read this post.

https://dev.to/coly010/clean-javascript-10-tips-37f4